benchmarks/results.md

```diff
@@ -0,0 +1,4 @@
+# Tue Apr 14 2015 12:36:47 GMT-0700 (PDT)
+Todo.create x 233 ops/sec ±100.80% (73 runs sampled)
```

Comment:
@raymondfeng I have no idea why this is `±100.80%` (relative margin or error). Does this mean it can be up to 100% wrong? lol